Makefile

```diff
@@ -77,13 +76,14 @@ list_merge_keys: ## List all GPG keys allowed to sign merge commits.
 .PHONY: globals
 PASSWORD_STORE_DIR?=${HOME}/.paas-pass
 globals:
-	$(eval export AWS_DEFAULT_REGION=${AWS_DEFAULT_REGION})
 	$(eval export PASSWORD_STORE_DIR=${PASSWORD_STORE_DIR})
 	@true
 
 .PHONY: dev
 dev: globals ## Set Environment to DEV
```

Comment:
is there is a reason why we don't call check-env here?

Reply:
Not particularly, It's checked lower down by all the other tasks that do actions (eg pipelines), so it's not strictly necessary here. It's also not something that's been changed in the PR.

I guess we could add it here for consistency with paas-bootstrap, but I'm not sure it's necessary.